sci-libs/silo/silo-4.11.ebuild

```diff
@@ -1,41 +1,43 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=8
 
-inherit epatch fortran-2
+inherit fortran-2
 
 DESCRIPTION="A mesh and field I/O library and scientific database"
 HOMEPAGE="https://wci.llnl.gov/simulation/computer-codes/silo"
-SRC_URI="https://wci.llnl.gov/content/assets/docs/simulation/computer-codes/${PN}/${P}/${P}.tar.gz"
+SRC_URI="https://wci.llnl.gov/sites/wci/files/2021-09/${P}-bsd.tgz"
+
 SLOT="0"
 LICENSE="BSD"
 KEYWORDS="~amd64 ~x86"
-IUSE="hdf5 +silex static-libs +qt5 test"
-RESTRICT="!test? ( test )"
+IUSE="browser +hdf5 +silex static-libs +qt5"
+
+# Test is actually broken in 4.11
+# Restricted for now
+RESTRICT="test"
 
 REQUIRED_USE="silex? ( qt5 )"
 
 RDEPEND="
 	hdf5? ( sci-libs/hdf5 )
-	qt5? ( dev-qt/qtgui:5 )"
+	qt5? ( dev-qt/qtgui:5 )
+"
+
 DEPEND="${RDEPEND}"
 
-src_prepare() {
-	epatch "${FILESDIR}/${P}-qtlibs.patch"
-	epatch "${FILESDIR}/${P}-qt5.patch"
-	epatch "${FILESDIR}/${P}-tests.patch"
-	epatch "${FILESDIR}/${P}-mpiposix.patch"
```

Comment:
Are you sure we don't need these patches anymore?

Reply:
No, I'm not, but actually I need to repatch them as they all fails to apply